enterprise/app/trends/trends_chart.tsx

```diff
@@ -77,7 +94,20 @@ export default class TrendsChartComponent extends React.Component {
                 />
               }
             />
-            <Bar yAxisId="primary" name={this.props.name} dataKey={this.props.extractValue} fill="#607D8B" />
+            <Bar
+              className={this.props.clickableBars ? "trends-clickable-bar-primary" : ""}
+              yAxisId="primary"
+              name={this.props.name}
+              dataKey={this.props.extractValue}
+              fill="#607D8B">
+              {this.props.data.map((date, index) => (
+                <Cell
+                  cursor={this.props.clickableBars ? "pointer" : "default"}
+                  key={`cell-${index}`}
+                  onClick={this.props.clickableBars ? this.handleClick.bind(this, date) : null}
+                />
+              ))}
+            </Bar>
             {this.props.extractSecondaryValue && this.props.secondaryLine && (
               <Line
```

Comment:
Is there a way to set `pointer-events: none` on this line? Sometimes it gets in the way of the bars and it causes flickering:

https://user-images.githubusercontent.com/2414826/128788440-0e8d4c25-393e-44eb-91bc-bdf4c052291f.mp4